src/components/content/policies/Policies.tsx

```diff
@@ -0,0 +1,272 @@
+
+    const addPoliciesManagementService = () => {
+        setId('');
+        setCurrentPolicyService(undefined);
+        setIsOpenAddOrUpdatePolicyModal(true);
+    };
+    const closeAddOrUpdatePoliciesManagementServiceModal = () => {
+        setIsOpenAddOrUpdatePolicyModal(false);
+        refreshPoliciesManagementServiceList();
+    };
+    const updatePoliciesManagementService = (record: PolicyVo) => {
+        setCurrentPolicyService(record);
+        setIsOpenAddOrUpdatePolicyModal(true);
+    };
+
+    const getCancelUpdateStatus = (isCancle: boolean) => {
+        if (isCancle) {
+            setIsOpenAddOrUpdatePolicyModal(false);
+            refreshPoliciesManagementServiceList();
+        }
+    };
+
+    return (
+        <>
+            <div className={'services-content policies-instance-list'}>
+                {deletePoliciesManagementServiceRequest.isSuccess && id.length > 0 ? (
+                    <PolicyDeleteResultStatus
+                        id={id}
+                        isError={deletePoliciesManagementServiceRequest.isError}
+                        isSuccess={deletePoliciesManagementServiceRequest.isSuccess}
+                        error={deletePoliciesManagementServiceRequest.error}
+                        getDeleteCloseStatus={getDeleteCloseStatus}
+                    />
+                ) : null}
+                <div className={'add-policy-container'}>
+                    <div className={'update-policy'}>
+                        <Button
+                            type='primary'
+                            onClick={refreshPoliciesManagementServiceList}
+                            icon={<SyncOutlined />}
+                            disabled={
+                                id.length > 0 &&
+                                (deletePoliciesManagementServiceRequest.isPending ||
+                                    deletePoliciesManagementServiceRequest.isSuccess ||
+                                    deletePoliciesManagementServiceRequest.isError)
+                            }
+                        >
+                            refresh
+                        </Button>
+                    </div>
+
+                    <div className={'add-policy'}>
+                        <Button
+                            onClick={addPoliciesManagementService}
+                            type='primary'
+                            icon={<PlusCircleOutlined />}
+                            disabled={
+                                id.length > 0 &&
+                                (deletePoliciesManagementServiceRequest.isPending ||
+                                    deletePoliciesManagementServiceRequest.isSuccess ||
+                                    deletePoliciesManagementServiceRequest.isError)
+                            }
+                        >
+                            add
+                        </Button>
+                    </div>
+                </div>
+
+                <Modal
```

Comment:
everytime i open the update modal, i see two warnings in console logs

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/bc0eb9f1-fd88-4e18-b998-2713f25a25ed)